src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -398,8 +404,19 @@ public DeletePageObject deletePage() {
     return new DeletePageObject(driver);
   }
 
-  public String getBannerNotificationText() {
-    return bannerNotification.getText();
+  public List<Notification> getNotifications(){
+    List<Notification> notificationList = new ArrayList<>();
+    for (WebElement notificationElement : notificationElements){
+      Notification notification = new Notification(driver, notificationElement);
+      notificationList.add(notification);
+    }
+    return notificationList;
+  }
+
+  public List<Notification> getNotifications(String notificationType){
```

Comment:
What do you think about renaming this method to be more accurate e.g. `getNotificationsListByType` ?